client-java/src/unit-tests/java/uk/gov/gchq/palisade/client/internal/download/DataRequestSerialisationTest.java

```diff
@@ -36,7 +35,7 @@ void setup() {
     @Test
     @DisplayName("Test DataRequest [De]Serialisation")
     void testDataRequestSerialisation() throws Exception {
-        DataRequest expected = createDataRequest(b -> b
+        DataRequest expected = DataRequest.createDataRequest(b -> b
             .token("blah")
```

Comment:
something a bit more specific?
```suggestion
            .token("test-request-token")
```

Reply:
Sorted